tor/core/inbox.py

```diff
@@ -93,7 +93,8 @@ def process_reply(reply, config):
 
         if (
             'done' in r_body or
-            'deno' in r_body  # we <3 u/Lornescri
+            'deno' in r_body or # we <3 u/Lornescri
+            'dibs' in r_body
```

Comment:
Putting it here would only allow it to finish a post, not claim one.

Reply:
Oops it started with "d" and fit the pattern so I just forgot basic logic 😅